CoEpi/common/NumberFormatters.swift

```diff
@@ -8,10 +8,10 @@ final class Number2MaxDecimalsFormatter: NumberFormatter {
 
     override init() {
         super.init()
-        numberStyle = .decimal
+        numberStyle = .none
         minimumFractionDigits = 0
-        maximumFractionDigits = 2
-        roundingMode = .halfUp
+        maximumFractionDigits = 0
+        roundingMode = .down
```

Comment:
Why not decimals anymore? Also, if this is really necessary, it's probably enough to convert the value to integer and not use a formatter.

Reply:
the style.decimal was responsible for the thousands comma formatting error (https://developer.apple.com/documentation/foundation/numberformatter/style/).
We cant just use int since we ned to support fractional numbers. This may be overkill, but I will leave that tip to you to decide. As it stands, I am unable to break the formatting and it is showing up correctly in the Xcode debug window.

Reply:
Can you describe what exactly was the problem with the formatting? Can it be solved using [groupingSeparator](https://developer.apple.com/documentation/foundation/numberformatter/1412157-groupingseparator), maybe?

Either way it's wrong to change this specific formatter to this.

Reply:
when using "decimal" it auto corrected 1000 to 1,000.

Reply:
You probably can remove the grouping separator using e.g. `groupingSeparator`. As said, this formatter shouldn't be used for this (name "Number2MaxDecimalsFormatter"). Otherwise, if you don't need formatting, don't use a formatter.

Reply:
removing it is proving to be difficult. It seems that a formatter is needed without a drastic reworking. I will pick a more appropriate formatter.

Reply:
Ok. Just create a new one.